neuralmonkey/dataset.py

```diff
@@ -13,155 +13,6 @@
 from neuralmonkey.readers.utils import Reader
 from neuralmonkey.readers.plain_text_reader import UtfPlainTextReader
 
-SERIES_SOURCE = re.compile("s_([^_]*)$")
```

Comment:
No, nemůžu tam zacházet s typem `Dataset` dřív než je definovanej, to je důvod, proč ty funkce, co mají dataset ve svých anotacích, musí být pod tím.